build.gradle

```diff
@@ -208,7 +208,8 @@ subprojects {
     }
 
     compileJava {
-        options.compilerArgs << '-Xlint:all' << '-Xlint:deprecation' << '-Werror' << '-Xlint:-processing' << '-parameters'
+        options.compilerArgs << '-Xlint:all' << '-Xlint:deprecation' << '-Xlint:-processing' << '-parameters'
+//        options.compilerArgs << '-Xlint:all' << '-Xlint:deprecation' << '-Werror' << '-Xlint:-processing' << '-parameters'
```

Comment:
Deprecation warnings stop the building process. Because SDK contains some methods that have `@Deprecated` annotation. But it's not a critical issue.